src/composables/accounts.ts

```diff
@@ -22,41 +23,44 @@ import {
 import { AE_FAUCET_URL } from '@/protocols/aeternity/config';
 import { buildSimplexLink } from '@/protocols/aeternity/helpers';
 
-const isIdxInitialized = ref(false);
-const protocolNextAccountIdx = ref<Partial<Record<Protocol, number>>>(
+let isIdxInitialized = false;
+const protocolNextAccountIdx = ref<ProtocolRecord<number>>(
   // Aeternity starts from 1 as we have 1 account as default
   PROTOCOLS.reduce((acc, protocol) => ({ ...acc, [protocol]: 0, [PROTOCOL_AETERNITY]: 1 }), {}),
 );
+const protocolLastUsedGlobalIdx: ProtocolRecord<number> = {};
```

Comment:
Maybe we can rename this from  `protocolLastUsedGlobalIdx` to `protocolLastActiveGlobalIdx` because last used it means when you make some operation using the account. We need the last active account!

Reply:
Makes sense. Changed...